scripts/versions_manager.py

```diff
@@ -72,7 +74,21 @@ def dump(self, config=False, priority=999):
                 lines = 'Package: {0}\nPin: version {1}\nPin-Priority: {2}\n\n'.format(package, self.versions[package], priority)
                 result.append(lines)
             else:
-                result.append('{0}=={1}'.format(package, self.versions[package]))
+                separator="=="
+                if self.ctype == "web":
+                    parsed_url = urlparse(package)
+                    query = parse_qs(parsed_url.query)
+                    sp = query.get('sp')
+                    if '.blob.core.' in package and sp:
+                        if len(sp) > 1:
+                            continue
+                        elif len(sp) == 1 and sp[0] != 'r' and sp[0] != 'rl':
```

Comment:
Can we just change from '==' to ' =='?
The logic to validate privilege maybe change in future.